vnext/Microsoft.ReactNative.Cxx/NativeModules.h

```diff
@@ -145,6 +166,13 @@
   "\" to the attribute:\n"                                                        \
   "    REACT_SYNC_METHOD(method, L\"" methodName "\")\n...\n"
 
+#define REACT_SHOW_EVENTEMITTER_SIGNATURES(eventName, signatures)                \
```

Comment:
>EVENTEMITTER [](http://example.com/codeflow?start=19&length=12)

Should we use the term EVENT instead of EVENTEMITTER here? Or the EVENTEMITTER is a better term here?